sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -198,3 +198,6 @@ case class StreamingExplainCommand(
     ("Error occurred during query planning: \n" + cause.getMessage).split("\n").map(Row(_))
   }
 }
+
+/** Mark that the command can be skipped from cache lookup */
+trait IgnoreCachedData {}
```

Comment:
can we move it to catalyst module and make it more general? I'm not sure if only commands need it
```
/**
 * A [[LogicalPlan]] operator that does not use the cached results stored in CacheManager.
 */
trait IgnoreCachedData extends LogicalPlan

```

Reply:
Moved to catalyst